Cargo.toml

```diff
@@ -32,6 +32,9 @@ all-features = true
 pinned-stable = "1.73.0"
 pinned-nightly = "nightly-2023-10-30"
 
+[package.metadata.playground]
+features = ["__internal_use_only_features_that_work_on_stable"]
```

Comment:
I expected this list to be longer. Do you not need to list all of the features you'd like enabled?

Reply:
Whoops, I see what's going on.